### PR TITLE
New Resource `alicloud_simple_application_server_custom_image`  New Datasource `alicloud_simple_application_server_custom_images`

### PR DESCRIPTION
**Pull request by super-eggs**
New Resource `alicloud_simple_application_server_custom_image`
New Datasource `alicloud_simple_application_server_custom_images`

### CODE REVIEW

File: alicloud/resource_alicloud_simple_application_server_custom_image_test.go [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+				Config: testAccConfig(map[string]interface{}{
+					"status": "UnShare",
+				}),
+				Check: resource.ComposeTestCheckFunc(testAccCheck(map[string]string{})),
+			},
+			{
+				ResourceName:      resourceId,
+				ImportState:       true,
+				ImportStateVerify: true, ImportStateVerifyIgnore: []string{"instance_id", "system_snapshot_id", "status"},
+			},
```
Comment (xinmu0426) [inline alicloud/resource_alicloud_simple_application_server_custom_image_test.go:62]: 这个开aone了么

Comment (xinmu0426) [reply alicloud/resource_alicloud_simple_application_server_custom_image_test.go:62]: 换行一下

Comment (super-eggs) [reply alicloud/resource_alicloud_simple_application_server_custom_image_test.go:63]: 开了

**Comment by arLevi** [comment]
is it available ? the official docs doesn't say anything about it

**Pull request merged**